### ISSUE TITLE
Send invitations to potential gathertown moderators for Satellite Activity

### ISSUE THREAD

**Issue by rabernat** [pangeo-data/openoceancloud#6]
As described in [this Discourse post](https://discourse.pangeo.io/t/seeking-moderators-for-openoceancloud-whitepaper-sprint-sept-16-17/1713), we need quite a few moderators for our proposed satellite activity. There don't seem to be many volunteers, so we will need to pound the pavement here.

We have the following slots to fill

- [x] 9/15 22 UTC - chelle gentemann (@cgentemann)
- [x] 9/16 01 UTC - Emilio Mayorga (@emiliom)
- [x] 9/16 04 UTC Paige Martin
- [x] 9/16 07 UTC - Guillaume Eynard-Bontemps (@guillaumeeb)
- [x] 9/16 11 UTC - Leslie Smith (@leslie-yoc)
- [x] 9/16 13 UTC _US EAST COAST 9am_ Erin Robinson 
- [x] 9/16 16 UTC - Sindre (@sseppola)
- [x] #12 - Andrea (@ALMcCurdy)
- [x] 9/16 22 UTC - Deepak Cherian (@dcherian)
- [x] 9/17 01 UTC - Aimee Barciauskas (@abarciauskas-bgse)
- [x] 9/17 04 UTC - Suman Shekhar (@Sumanshekhar17)
- [x] 9/17 07 UTC - Tina Odaka (@tinaok)
- [x] 9/17 9 UTC - Leslie Smith (@leslie-yoc)
- [x] 9/17 11 UTC - Guillaume Eynard-Bontemps (@guillaumeeb)
- [x] 9/17 13 UTC _US EAST COAST 9am_  Ryan Abernathey      
- [x] 9/17 16 UTC Chelle Gentemann (@cgentemann)

The Google Doc has some specific name suggestions, but I didn't want to post these publicly.

Let's use this issue to track progress on inviting / confirming moderators.

**Comment by leslie-yoc** [comment]
I am happy to sign up for one of these slots. If you're struggling to get folks early in the morning US I can help out starting at 4:30/5am ET (8:30/9 UTC) and go until the 9am ET (13 UTC) slot. The only problem is 6:15-7:15 I'll be getting the kiddos ready for school. But I could have the laptop on in kitchen in case someone logs in?

**Comment by ALMcCurdy** [comment]
I went to add my name to 10am ET but was unable - if it turns out you need this please let me know

**Comment by rabernat** [comment]
> I went to add my name to 10am ET but was unable 

That's because you hadn't yet been invited to the @pangeo-data/openoceancloud team b/c I didn't know your GitHub username. I just invited you! 😉

**Comment by ALMcCurdy** [comment]
Thanks…

**Comment by leslie-yoc** [comment]
> I am happy to sign up for one of these slots. If you're struggling to get folks early in the morning US I can help out starting at 4:30/5am ET (8:30/9 UTC) and go until the 9am ET (13 UTC) slot. The only problem is 6:15-7:15 I'll be getting the kiddos ready for school. But I could have the laptop on in kitchen in case someone logs in?

@rabernat would you like for me to sign up for the timeslots 4:30-9am ET on 9/16 & 9/17? I'm trying to get my schedule set for those days in terms of availability for kid dropoff duty. I am happy to do it, I just want to get it on my calendar.

**Comment by rabernat** [comment]
> would you like for me to sign up for the timeslots 4:30-9am ET on 9/16 & 9/17?

Sure, that would be great! Just edit the top comment

![image](https://user-images.githubusercontent.com/1197350/131730256-fe6692f7-f8e3-48b2-a11c-20ac8c431538.png)

**Comment by rabernat** [comment]
Note that we now have 11 potential moderators over in https://discourse.pangeo.io/t/seeking-moderators-for-openoceancloud-whitepaper-sprint-sept-16-17/1713 (people who have liked the post).

@leslie-yoc - would you be able to help figure out how to match up these people with the open timeslots?

**Comment by leslie-yoc** [comment]
Yes, I will send those 11 a message via github and get them filled into the timeslot. More on that in a little bit.

**Comment by guillaumeeb** [comment]
OK, found the link in gitter. I took the only slot available for me, but if this helps I could also have taken the 9/17 9 and 13 slots. I also notice that the slots at 7 UTC only last 2hours, I can continue until 10 if needed (or more if this is easier for @leslie-yoc).

**Comment by leslie-yoc** [comment]
Hi @guillaumeeb!
Fantastic to have someone on the other side of the Atlantic take these slots! Not to eat up too much of your day, but how would you feel about going from 7-9 UTC and then 10-11UTC? I can get on at 9 (5am my time), but then from 10-11UTC (6-7am my time) I need to be making my kids breakfast and getting them off to school. I was going to have the laptop up in the kitchen, but would be so much easier to have someone else in charge of that time slot. It would actually probably be closer to 11:15/20 that I could get back on. That ok? Thanks so much!

**Comment by guillaumeeb** [comment]
@leslie-yoc yep I can do 10-11 UTC (12-13 my time) on Thursday 16!

**Comment by sseppola** [comment]
Thanks for the email @leslie-yoc!

I'm in CET-1 so 9/16 16 UTC is a reasonable hour for me outside of the tasks that are already assigned.

**Comment by Sumanshekhar17** [comment]
Thanks for the mail @leslie-yoc . I am up for 4 UTC on 9/17  :)

**Comment by dcherian** [comment]
I can do 9/16 22 UTC (16 MT)

**Comment by emiliom** [comment]
I can do 9/16 01 UTC (18 PT). Not sure if communicating this here is enough, but I've also gone ahead and "liked" the discourse post.

**Comment by abarciauskas-bgse** [comment]
Sorry for the late follow up (@cgentemann mentioned this opportunity to me last week) I can do 09/17 01 UTC (which I believe to be 09/16 18 PT) which is a slot that appears to still need a moderator.

**Comment by rabernat** [comment]
Great thanks Aimee! @leslie-yoc, can you forward Aimee the moderator onboarding materials?

**Comment by leslie-yoc** [comment]
Hi @abarciauskas-bgse you should have just received a calendar invite from my Columbia email address with your moderator timeslot and instructions. Please let us know if you have any questions! Also, yay rock climbing!

**Comment by sseppola** [comment]
@rabernat @leslie-yoc can you send me the onboarding materials? I've DMed my email in the Pangeo discourse

**Comment by leslie-yoc** [comment]
Sent! You should have received a calendar invite from my Columbia email address. Instructions are in the invite.

**Comment by rabernat** [comment]
I'm wondering if anyone happens to be free to moderate 10-11am. I'm scheduled to moderate but a conflict has come up.

**Comment by tinaok** [comment]
You mean 14UTC - 15UTC  (=16CET- 17 CET)?
If so I can take that.

**Comment by rabernat** [comment]
Yes I believe that is correct. Thanks @tinaok!

**Comment by rabernat** [comment]
I am actually running the whole thing inside a docker container like this:

```
git checkout https://github.com/rabernat/intro_to_physical_oceanography.git
docker run -it --rm --volume intro_to_physical_oceanography:/intro_to_physical_oceanography -p 8888:8888 pangeo/pangeo-notebook:latest /bin/bash -c "mamba install --yes -n notebook -c conda-forge rise ipytree nbgitpuller==1.0.2 notebook=6.4.4; jupyter lab --ip 0.0.0.0 /intro_to_physical_oceanography
```
